backend/src/main/java/com/benine/backend/database/MySQLDatabase.java

```diff
@@ -107,6 +208,9 @@ public void addPreset(Preset preset) {
       statement = connection.createStatement();
       String sql = preset.createAddSqlQuery();
       statement.executeUpdate(sql);
+      for (String tag : preset.getTags()) {
```

Comment:
You add the tags here to the database, but I think the deletion of presets will not work. When you delete a preset these tags has first to be removed before you can delete the preset because of the foreign key constraints.